ui/src/app/components/object-explorer/object-explorer.component.scss

```diff
@@ -159,3 +160,6 @@
 ::ng-deep .column-left .mat-tab-header-pagination-after {
   margin-right: 40px;
 }
+.blue-font {
```

Comment:
Since this is the generic font colour to be used across all the files, it makes sense to move it to styles.css

Reply:
Moved to global css as blue-font-color